### PR TITLE
Added links to Forum support docs

### LINKED ISSUE #460: Edit "How To Get Support" Content
## Description
We have a "How To Get Support" button on the Homepage.
It should guide the users to a page describing the different types of support (coding support, support for Hive etc.) and how to get this support.

## Close Criteria
- [ ] Create it as an index.md page anywhere in docs.
- [ ] Describe the different types of user support we provide
- [ ] Add Buttons with links to the correct pages where people can get support (Hive Forum #help etc.).
- [ ] We will link the page on the homepage once this issue is closed.

### PR DESCRIPTION
**Pull request by aidanheerdegen**
## Description

Added links to Forum support docs in user support section.

Fixes #460

## Type of change

Please delete options that are not relevant.

- [x] New link / content

## Checklist:

- [x] The new content is accessible and located in the appropriate section
- [x] My changes do not break navigation and do not generate new warnings
- [x] I have checked that the links are valid and point to the intended content
- [x] I have checked my code/text and corrected any misspellings
- [ ] I have chosen the correct tag for the level of [support provided](https://access-hive.org.au/#support)

When your pull request is ready please [request a review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review). 

Unless there is a specific person you want to review your PR please select the **reviewers team**: `ACCESS-Hive/reviewers`. This ensures the load for reviewing pull requests is shared equitably.

### CODE REVIEW

**Comment by aidanheerdegen** [comment]
@atteggiani I can't see the changes in the preview because the support page is no longer in the navigation structure. Can you advise how I should proceed.

**Comment by aidanheerdegen** [comment]
@atteggiani can you take a look and let me know if it makes sense to you.

File: docs/about/user_support.md
[the remark this answers is not in the document]
Comment (heidinett) [reply docs/about/user_support.md:1]: If you want a stickler's opinion, I would add a comma in the following two places:
1. For resources developed by another party that ACCESS-Hive links to**,** there should be information on how to get help on the linked site.
2. In the case of ACCESS-NRI supported documentation and software**,** please ask on the [ACCESS-Hive Forum](https://forum.access-hive.org.au/).

**Comment by atteggiani** [comment]
Merging for now, for external review, but will keep an issue open to deal with the page.

**Pull request merged**